### ISSUE TITLE
Process.title

### ISSUE THREAD

**Issue by alexkavon** [modubot/modubot.js#55]
Add a config option for naming the bot and use a lowercase version of that name for the process.title in nodejs

**Comment by kamaln7** [comment]
@alexkavon does #59 work?

**Comment by alexkavon** [comment]
That works, however I was thinking of something like:

config.yml
~~~
name: modubot
~~~

Bot.ts
~~~
process.title = config.name
~~~

Or something....

**Comment by kamaln7** [comment]
That would be one more config option that you'll need to edit. I think irc server + nick should be unique enough. +@reynir thoughts?

**Comment by reynir** [comment]
I would use either just the nick, or possibly the realname. I can't really think of a situation where you wouldn't want it to be one of those.

**Comment by kamaln7** [comment]
Cool. The server is there just so you can differentiate between the different processes if you have a Modubot instance with the same nick connected to two+ different servers.

**Comment by alexkavon** [comment]
or in Bot.ts

~~~
process.title = this.config.name || 'modubot';
~~~

kamal_ learn 2 default configs pl0x

**Comment by kamaln7** [comment]
https://github.com/modubot/modubot.js/tree/master/config

Modubot falls back the default config file if a key is missing from the config.yml file.

**Comment by alexkavon** [comment]
k

**Comment by alexkavon** [comment]
Your PR looks good then.

**Comment by kamaln7** [comment]
Merged #59
